squirrel_object_perception/src/squirrel_fuse_lumps_static_octomap.cpp

```diff
@@ -67,33 +66,36 @@ bool FuseLumpsIntoOctomap::createOctomapWithLumpsCB (squirrel_object_perception_
             octomap::OcTreeKey key = it.getKey();
             octomap::OcTreeNode* node = fused_map->search(key);
             if (node==NULL) {
-                std::cout << "(" << it.getCoordinate().x() << "," << it.getCoordinate().y() << ") Check node" << std::endl;
                 fused_map->setNodeValue(key,octomap::logodds(fused_map->getClampingThresMax()));
             } else if (!fused_map->isNodeOccupied(node)) {
-                std::cout << "(" << it.getCoordinate().x() << "," << it.getCoordinate().y() << ") Check node" << std::endl;
                 (*it).setLogOdds(octomap::logodds(fused_map->getClampingThresMax()));
             }
         }
     }
     return true;
 }
 
-bool FuseLumpsIntoOctomap::receiveOctomapWithLumpsCB (squirrel_object_perception_msgs::ReceiveOctomapWithLumpsRequest &request, squirrel_object_perception_msgs::ReceiveOctomapWithLumpsResponse &response) {
-    if (fused_map != NULL && fused_map->getNumLeafNodes() > 0) {
-        octomap_msgs::Octomap map_msg;
-        map_msg.header.stamp = ros::Time::now();
-        map_msg.header.frame_id = "/map";
-        if (octomap_is_binary) {
-            octomap_msgs::binaryMapToMsg(*fused_map, map_msg);
-        } else {
-            octomap_msgs::fullMapToMsg(*fused_map, map_msg);
-        }
-        response.fusedOctomap = map_msg;
-        octomap_pub.publish(map_msg);
-        return true;
+bool FuseLumpsIntoOctomap::receiveOctomapWithLumpsCB (octomap_msgs::GetOctomapRequest &request, octomap_msgs::GetOctomapResponse &response) {
```

Comment:
@edith-langer  I get the error: error: 'bool FuseLumpsIntoOctomap::receiveOctomapWithLumpsCB' is not a static member of 'class FuseLumpsIntoOctomap'

Reply:
I forgot to commit the header file...